src/Donations/PaymentMethods/PaymentFunctions.ts

```diff
@@ -512,13 +521,7 @@ export async function handleStripeSCAPayment({
           payment_method: {
             billing_details: buildBillingDetails(contactDetails),
           },
-          return_url: `${
-            window.location.origin
-          }/?context=${donationID}&method=Giropay&tenant=${tenant}&country=${country}&locale=${
-            localStorage.getItem("language")
-              ? localStorage.getItem("language")
-              : "en"
-          }`,
+          return_url: `${window.location.origin}/?context=${donationID}&method=Giropay&tenant=${tenant}&country=${country}`,
```

Comment:
I am not sure why you did not append `&locale=${i18n.language}` to the URL here while you do it at other places?

Reply:
Missed this out, will update.

Reply:
Fixed in 2cfc02e5dc3f0872b98c331fe5b99c849e684fe0